2-mandatory/3-financial-times.js

```diff
@@ -23,32 +62,44 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  // TODO
+    
+    return allArticleTitles.filter(article => /\d/.test(article));
```

Comment:
Nice use of a regex here, they're a really useful tool if you can get past the intial complexity of them